rabbitmq/tox.ini

```diff
@@ -16,29 +16,16 @@ deps =
 passenv =
     COMPOSE*
     DOCKER*
-
-[testenv:unit]
-commands =
-    pip install --require-hashes -r requirements.txt
-    pytest -v -m"unit"
-
-[testenv:rabbitmq3.5.0]
-setenv = RABBITMQ_VERSION=3.5.0
-commands =
-    pip install --require-hashes -r requirements.txt
-    pytest -v -m"not unit"
-
-[testenv:rabbitmq3.6.0]
-setenv = RABBITMQ_VERSION=3.6.0
-commands =
-    pip install --require-hashes -r requirements.txt
-    pytest -v -m"not unit"
-
-[testenv:rabbitmq3.7.0]
-setenv = RABBITMQ_VERSION=3.7.0
+setenv =
+    3.5.0: RABBITMQ_VERSION=3.5.0
+    3.6.0: RABBITMQ_VERSION=3.6.0
+    3.7.0: RABBITMQ_VERSION=3.7.0
 commands =
     pip install --require-hashes -r requirements.txt
-    pytest -v -m"not unit"
+    unit: pytest -v -m"unit"
+    3.5.0: pytest -v -m"not unit"
```

Comment:
Couldn't this just be one line instead of three, without the version number in front?

Reply:
no because of the unit testing

Reply:
unless there's some kind of if/else in tox but I don't think there is as far as I've seen